cmd/plan_test.go

```diff
@@ -0,0 +1,116 @@
+package cmd
+
+import (
+	"fmt"
+	"github.com/dpastoor/goutils"
+	"github.com/spf13/afero"
+	//"os"
+	"github.com/stretchr/testify/assert"
+	"path/filepath"
+	"testing"
+)
+
+func InitializeTestEnvironment(goldenSet, testName string) afero.Fs{
```

Comment:
check the suite https://github.com/stretchr/testify subpackage of testify, which I think is does a lot of these things for you.

Reply:
I'll need to leave a good chunk of InitializeTestEnvironment as it is, but I have started to implement the setup and teardown functions, and that will definitely be helpful for future testing efforts.